src/main/java/frc/robot/Robot.java

```diff
@@ -56,9 +56,12 @@ public void robotInit() {
 
     robotContainer = new RobotContainer();
 
-    chooser.setDefaultOption("Taxi", Constants.Auton.TAXI); // default is taxi mode
+    // TODO: change to correct default
+    chooser.setDefaultOption(
+        "Test Mode", Constants.Auton.TEST); // default is TEST mode for now, change later
```

Comment:
Default is two ball in competition branch.  Does it make sense to update our default here to what we think we should do in competition?  Just to reduce competition day changes?

Reply:
I will change that. The default auton mode will probably be three ball or four ball depending on what auton mode turns out best after testing is done.